week-6/Homework/mandatory/2-quotegenerator/style.css

```diff
@@ -1 +1,28 @@
 /** Write your CSS in here **/
+
+body {
+    background-color: #F9B103;
+    height: 100vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+div {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-end;
+    background-color: white;
+    padding: 1.5em 3em 1.5em 3em;
+    margin: 5em;
```

Comment:
try to keep the button place constant, the UX is not nice when the user needs to look for the button each time